pom.xml

```diff
@@ -871,9 +880,32 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <argLine>--add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.util.regex=ALL-UNNAMED --add-opens java.base/java.util.stream=ALL-UNNAMED --add-opens java.base/java.net=ALL-UNNAMED</argLine>
+                    <argLine
+                    >@{argLine} --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED
```

Comment:
why adding @{argLine} ?

Reply:
Black magic needed, I took it from the doc: https://www.eclemma.org/jacoco/trunk/doc/prepare-agent-mojo.html

Reply:
👌🏻